docs/admin-guide.md

```diff
@@ -0,0 +1,193 @@
+
+### Manual Installation
+
+If your server doesn't have access to the internet, you can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-nagios/releases) and upload it to your server via **System Console > Plugin Management**. The releases on this page are the same used by the Marketplace. To learn more about how to upload a plugin, see [the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
+
+### Cloud
+
+### Upgrade
+
+Ensure you configure the plugin before you enable it.
+
+## Configuration
+
+### Enter the URL for your Nagios instance
+
+1. In Mattermost, go to **System Console > Plugins > Nagios**.
+2. Set the **Nagios URL**. Remember to add `http://` or `https://` at the beginning (e.g.: `https://nagios.fedoraproject.org`).
+3. Set **Nagios Username** to the Nagios user used to authentiacte.
+4. Sett **Nagios Password** to the password of that user.
+5. Select **Save**.
+6. Go to **Plugins > Nagios**, then select **Enable**.
+7. The plugin is now ready to use.
+
+### Configure the configuration files watcher
+
+*This step is optional, although highly recommended.*
+
+Regenerate the token for the configuration files watcher:
+1. In Mattermost, go to **System Console > Plugins > Nagios**.
+2. Select **Regenerate** to regenerate the token.
+3. Copy the token - you're going to use it later.
+4. Select **Save**.
+5. Switch to the machine where Nagios is running:
+  - Download the latest stable version of the watcher from the [releases page](https://github.com/mattermost/mattermost-plugin-nagios/releases)
+  - Move the watcher: `chmod +x watcher1.1.0.linux-amd64 && sudo mv watcher1.1.0.linux-amd64 /usr/local/bin/watcher`.
+
+### Run the watcher as a systemd service
+
+#### Prepare the systemd service unit file
+
+Prepare the systemd service unit file by adjusting `dir` (default if not set: `/usr/local/nagios/etc/`), `url`, and `token` flags to your setup.
+
+```sh
+sudo bash -c 'cat << EOF > /etc/systemd/system/mattermost-plugin-nagios-watcher.service
+[Unit]
+Description=Nagios configuration files monitoring service
+After=network.target
+
+[Service]
+Restart=on-failure
+ExecStart=/usr/local/bin/watcher -dir /nagios/configuration/files/directory -url https://mattermost.server.address/plugins/nagios -token TheTokenFromStep1
+
+[Install]
+WantedBy=multi-user.target
+EOF'
+```
+
+#### Start the watcher
+
+```sh
+systemctl daemon-reload
+systemctl enable mattermost-plugin-nagios-watcher.service
+systemctl start  mattermost-plugin-nagios-watcher.service
+```
+
+### Get logs
+![logs](/docs/images/logs.png)
+
+### Receive system monitoring reports
+![reports](/docs/images/reports.png)
+
+### Receive notifications about changes to the configuration
+![changes](/docs/images/changes.png)
+
+Ultimately, this will make you or your team more productive and make the experience with Nagios smoother.
+
+
+## Web Hooks
+
+## Slash Commands 
```

Comment:
These slash commands are duplicated from the README. I'm thinking they should stay in the README, since these are the only "end-user" features of the plugin, even if they are only accessible to admins. Maybe we should just link to the README here, specifically to the "Slash Commands" section